lib/hawkular/metrics/metric_api.rb

```diff
@@ -129,6 +129,19 @@ def get_data(id, starts: nil, ends: nil, bucketDuration: nil, buckets: nil, perc
         resp.is_a?(Array) ? resp : [] # API returns no content (empty Hash) instead of empty array
       end
 
+      # Retrieve raw data for multiple metrics.
+      # This API is experimental and may undergo non-backwards compatible changes on the future
+      # @param ids [Array[String]] metric definition ids
+      # @param starts [Integer] optional timestamp (default now - 8h)
+      # @param ends [Integer] optional timestamp (default now)
+      # @param limit [Integer] optional limit the number of data points returned
+      # @param order [String] optional Data point sort order, based on timestamp (ASC, DESC)
+      # @return [Array[Hash]] named datapoints
+      def raw_data(ids, starts: nil, ends: nil, limit: nil, order: nil)
```

Comment:
Being on the file metric_api and Metrics class, should we be verbose enough to refer to it as metric_ids ? 
I don't have any problem on changing them. I was just trying to mimic the params on get_data[1] method. Almost the same params can be used on both methods and I don't see why we should name 'limit' differently on this method as this could lead to confusions.

Should I change `ids` to `metric_ids` and `limit` to `max_data_points`?

[1] https://github.com/josejulio/hawkular-client-ruby/blob/2fe78d22f13cf45cf611a4e9cf859eb49575c921/lib/hawkular/metrics/metric_api.rb#L123